api/server/user/data.go

```diff
@@ -224,6 +225,9 @@ func (s *Service) CidInfo(ctx context.Context, req *userPb.CidInfoRequest) (*use
 		}
 		res = append(res, cidInfo)
 	}
+	sort.Slice(res, func(a, b int) bool {
+		return res[a].Cid < res[b].Cid
```

Comment:
Just sorting by cid so the results are in a consistent order. There's nothing better to sort by really.